packages/mask/src/extension/popups/hook/useLoadMore.ts

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect, type RefObject } from 'react'
+import { throttle } from 'lodash-es'
+export function useLoadMore(ref: RefObject<HTMLElement>) {
```

Comment:
CC @UncleBill  Do we have any preexist solution?

Reply:
We do.
@beyond009 You can take a look at this:

https://github.com/DimensionDev/Maskbook/blob/dd35db783d5b6446107242b8c1040e51a0c5cab9/packages/mask/src/extension/popups/pages/Wallet/components/ActivityList/index.tsx#L78-L80